src/index.js

```diff
@@ -1,5 +1,3 @@
 import statsPlugin from './stats';
 
-export default function stats(emitter, upstreamName) {
-  return statsPlugin.bind(this, emitter, upstreamName);
-}
+export default statsPlugin;
```

Comment:
What purpose is this file serving?

Reply:
LMAO absolutely no purpose! My brain has stopped working 🙈

Reply:
Each plugin expect a `context` object as first parameter and the `next` function as second parameter which is injected by `http-transport`. If you want to pass extra information to the plugin (in this case the emitter and the `upstreamName`) you have to bind them to the function, and will be prefixed in the `args` of the function itself. This "index" purpose served that purpose.

Reply:
We definitely won't need the emitter given we are going to read the cache status from the context. So the question is, are we getting rid of the `upstreamName` parameter?

Reply:
Yep, I just removed that to rule out it was a weird even binding thing that was causing issues. I refactored the `stats` module to be more like the other http-transport plugins, but happy to put it back to how it was before 👍

Reply:
If we are returning a closure that has the same behaviour, I remember I did this for testing purposes. As long as it is easily testable I'm fine with this approach, which means we can remove the index and we should update the `rollup` config.

Reply:
Cool, yeah tests seem fine 👍